saleor/cart/utils.py

```diff
@@ -0,0 +1,19 @@
+from satchless.item import InsufficientStock
+
+
+def has_available_products(cart):
```

Comment:
I think `contains_unavailable_products` would be a clearer indication of the intent.